open_humans/templates/member/my-member-connected-data.html

```diff
@@ -60,26 +66,14 @@
         {% endfor %} {# data_file in data_files #}
       </table>
     {% else %}
-      <p>There are no files associated with this source.</p>
+      <p>There are not currently any files associated with this source.</p>
```

Comment:
Super weird phrasing. Should probably be `There are currently no files associated with this source`.